wordpress-mobile/gutenberg-mobile
# [FIX] inserter

This PR fixes an inserter point since it is rendered even if shouldn't

Gutenberg PR - https://github.com/WordPress/gutenberg/pull/20195

To test:
- Remove all from initial html
- Click Add Block button
- The inserter should be shown instead of paragraph placeholder - below the title
- Add group block
- Add a block in that group
- The inserter shouldn't be visible after that action and should be in the right place when the bottom sheet is open (please swipe bottom sheet a bit to see if inserter is in the right place)
- Click and hold on the Add Button (long press) and chose `add before` or `add after` and check if inserter is in the right place


PR submission checklist:
